packages/flutter/test/material/scrollbar_test.dart

```diff
@@ -1056,7 +1056,6 @@ void main() {
     await tester.pump();
     await tester.pump(const Duration(milliseconds: 200));
     expect(find.byType(CupertinoScrollbar), paints..rrect());
-    final CupertinoScrollbar scrollbar = find.byType(CupertinoScrollbar).evaluate().first.widget as CupertinoScrollbar;
-    expect(scrollbar.controller, isNotNull);
+    final CupertinoScrollbar scrollbar = tester.widget<CupertinoScrollbar>(find.byType(CupertinoScrollbar));    expect(scrollbar.controller, isNotNull);
```

Comment:
It looks like a line break was left out here

Reply:
Ah my IDE is unforgivable today. :P